src/components/ProductImage/index.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import './style.css';
+import Plus from 'assets/icons/Plus.svg';
+import RegisterButton from 'components/RegisterButton';
+
+const UploadImage = () => {
+  const [productImage, setProductImage] = useState(null);
+  const [productName, setProductName] = useState('');
+  const [productIntroduction, setProductIntroduction] = useState('');
+  const [productPrice, setProductPrice] = useState('');
+  const [productTag, setProductTag] = useState('');
+
+  const handleImageChange = (e) => {
+    const file = e.target.files[0];
+    if (file && file.type.startsWith('image/')) {
+      setProductImage(URL.createObjectURL(file));
+    }
+  };
+
+  const isFormValid = () => {
+    return (
+      productName.trim() &&
+      productIntroduction.trim() &&
+      productPrice.trim() &&
+      productTag.trim()
+    );
+  };
+
+  return (
+    <form onSubmit={(e) => e.preventDefault()} className='form-container'>
+      <div>
+        <label className='product-image-title'>상품 이미지</label>
+        <div className='image-box'>
+          {productImage ? (
+            <img
+              src={productImage}
+              alt='상품 이미지'
+              className='uploaded-image'
+            />
+          ) : (
+            <>
+              <img src={Plus} alt='이미지 추가' />
+            </>
+          )}
+          <input
+            type='file'
+            accept='image/*'
+            onChange={handleImageChange}
+            style={{ display: 'none' }}
+            id='image-upload'
+          />
+          <label htmlFor='image-upload' className='image-upload-label'>
+            {productImage ? '이미지 변경' : '이미지 등록'}
+          </label>
+        </div>
+      </div>
+      <div>
+        <label className='product-name'>상품명</label>
+        <input
+          type='text'
+          name='productName'
+          placeholder='상품명을 입력해주세요'
+          className='product-name-input'
+          value={productName}
+          onChange={(e) => setProductName(e.target.value)}
+        />
+      </div>
+      <div>
+        <label className='product-introduction'>상품 소개</label>
+        <textarea
+          type='text'
+          name='productIntroduction'
+          placeholder='상품 소개를 입력해주세요'
+          className='product-introduction-input'
+          value={productIntroduction}
+          onChange={(e) => setProductIntroduction(e.target.value)}
+        ></textarea>
```

Comment:
children 이 없는 태그일 경우에는 self-closing 태그 사용해도 좋아용!

```suggestion
        <textarea
          type='text'
          name='productIntroduction'
          placeholder='상품 소개를 입력해주세요'
          className='product-introduction-input'
          value={productIntroduction}
          onChange={(e) => setProductIntroduction(e.target.value)}
        />
```

Reply:
참고해서 이번 스프린트 미션 커밋 때 반영하겠습니당!!